File: Bab 8/8.2.1 Fill Array of Integer.c
#include <stdio.h>

int main()
{
	
	int arrayInteger[5], penghitung;

	for (penghitung = 0; penghitung < 5; penghitung++)
	{
		arrayInteger[penghitung] = penghitung;
	}

	for (penghitung = 0; penghitung < 5; penghitung++)
	{
		printf("%d\n", arrayInteger[penghitung]);
	}	

	return 0;
}

File: Bab 8/8.2.1 Fill Array of Integer.java
/*
 *  Name Program    : Fill Array of Integer
 *  Purpose         : Try Code from Logic Algorithm and Basic Programming Book chapter 8
 *  Author          : Yogi Siswanto
 *  Email           : yogisiswanto.c2@gmail.com
 *  Date Created    : 21th April 2021
 *  Language        : Java Language
 */

public class Main{

	// Main function
	public static void main(String[] args) {
		
		// Variable declaration
		int[] arrayInteger = new int[5]; 
		int penghitung;

		// Loop for fill array of integer
		for (penghitung = 0; penghitung < 5; penghitung++)
		{
			arrayInteger[penghitung] = penghitung;
		}

		// Loop for print to console value from array of integer
		for (penghitung = 0; penghitung < 5; penghitung++)
		{
			System.out.println(arrayInteger[penghitung]);
		}	
	}
}